### PR TITLE
Add sysfs led indicator test into automated test coverage(New)

### PR DESCRIPTION
**Pull request by rickwu666666**
## Description
Add led indicator test auto-test job and plan. Also, add it into ce-oem auto test coverage due to we have a [regression issue](https://bugs.launchpad.net/shiner/+bug/2057719) found in the post-gm stage.

## Resolved issues
N/A

## Tests
Sideload results with make up some non-existent led paths
https://certification.canonical.com/hardware/202205-30270/submission/361748/test-results/

### CODE REVIEW

File: providers/base/units/led/jobs.pxu [hunk 1]
```diff
@@ -349,6 +349,23 @@ requires: manifest.has_led_indicator == 'True'
...
+  Automated check that {name} LED turns on and off.
+plugin: shell
+user: root
+flags: also-after-suspend
+estimated_duration: 10
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_led_indicator == 'True'
+command:
+    led_control_test.py -n {path} -d 10
+
```
Comment (kissiel) [inline providers/base/units/led/jobs.pxu:368]: The job doesn't check if LED turns on and off.

**Review by kissiel — CHANGES_REQUESTED** [review]
The job doesn't check if LED turns on and off.

**Comment by rickwu666666** [comment]
> The job doesn't check if LED turns on and off.

Hi @kissiel,
Yes, you are correct, this job is not able to tell if the LED actually is on or off. However, the script checks whether the value is expected or not after writing the value to the node.  Wondering if that is good enough.
https://github.com/canonical/checkbox/blob/main/providers/base/bin/led_control_test.py#L61

File: providers/base/units/led/jobs.pxu (hunk 1 above)
Comment (rickwu666666) [reply providers/base/units/led/jobs.pxu]: Hi @kissiel,
Yes, you are correct, this job is not able to tell if the LED actually is on or off. However, the script checks whether the value is expected or not after writing the value to the node. Wondering if that is good enough.
https://github.com/canonical/checkbox/blob/main/providers/base/bin/led_control_test.py#L61

**Pull request closed without merging**